images/linux/scripts/SoftwareReport/SoftwareReport.Generator.ps1

```diff
@@ -0,0 +1,212 @@
+    (Get-AzCopy10Version),
+    (Get-BazelVersion),
+    (Get-BazeliskVersion),
+    (Get-CMakeVersion),
+    (Get-CurlVersion),
+    (Get-DockerComposeVersion),
+    (Get-DockerMobyVersion),
+    (Get-DockerBuildxVersion),
+    (Get-GitVersion),
+    (Get-GitLFSVersion),
+    (Get-GitFTPVersion),
+    (Get-GoogleCloudSDKVersion),
+    (Get-HavegedVersion),
+    (Get-HerokuVersion),
+    (Get-HHVMVersion),
+    (Get-SVNVersion),
+    (Get-JqVersion),
+    (Get-KindVersion),
+    (Get-KubectlVersion),
+    (Get-KustomizeVersion),
+    (Get-LeiningenVersion),
+    (Get-M4Version),
+    (Get-HGVersion),
+    (Get-MinikubeVersion),
+    (Get-NewmanVersion),
+    (Get-NvmVersion),
+    (Get-PackerVersion),
+    (Get-PhantomJSVersion),
+    (Get-SwigVersion),
+    (Get-TerraformVersion),
+    (Get-UnZipVersion),
+    (Get-WgetVersion),
+    (Get-ZipVersion),
+    (Get-ZstdVersion)
+)
+
+if (-not (Test-IsUbuntu16)) {
+    $toolsList += @(
+        (Get-PodManVersion),
+        (Get-BuildahVersion),
+        (Get-SkopeoVersion)
+    )
+}
+
+$markdown += New-MDList -Style Unordered -Lines ($toolsList | Sort-Object)
+
+$markdown += New-MDHeader "CLI Tools" -Level 3
+$markdown += New-MDList -Style Unordered -Lines @(
+    (Get-AlibabaCloudCliVersion),
+    (Get-AWSCliVersion),
+    (Get-AWSCliSessionManagerPluginVersion),
+    (Get-AWSSAMVersion),
+    (Get-AzureCliVersion),
+    (Get-AzureDevopsVersion),
+    (Get-GitHubCliVersion),
+    (Get-HubCliVersion),
+    (Get-NetlifyCliVersion),
+    (Get-OCCliVersion),
+    (Get-ORASCliVersion),
+    (Get-VerselCliversion)
+)
+
+$markdown += New-MDHeader "Java" -Level 3
+$markdown += Get-JavaVersions | New-MDTable
+$markdown += New-MDNewLine
+
+$markdown += New-MDHeader "PHP" -Level 3
+$markdown += Build-PHPTable | New-MDTable
+$markdown += New-MDNewLine
+
+$markdown += New-MDHeader "Haskell" -Level 3
+$markdown += New-MDList -Style Unordered -Lines @(
+    (Get-GHCVersion),
+    (Get-CabalVersion),
+    (Get-StackVersion)
+)
+
+$markdown += New-MDHeader "Rust Tools" -Level 3
+$markdown += New-MDList -Style Unordered -Lines @(
+    (Get-RustVersion),
+    (Get-RustupVersion),
+    (Get-RustdocVersion),
+    (Get-CargoVersion)
+)
+
+$markdown += New-MDHeader "Packages" -Level 4
+$markdown += New-MDList -Style Unordered -Lines @(
+    (Get-BindgenVersion),
+    (Get-CargoAuditVersion),
+    (Get-CargoOutdatedVersion),
+    (Get-CargoClippyVersion),
+    (Get-CbindgenVersion),
+    (Get-RustfmtVersion)
+)
+
+$markdown += New-MDHeader "Browsers and Drivers" -Level 3
+$markdown += New-MDList -Style Unordered -Lines @(
+    (Get-ChromeVersion),
+    (Get-ChromeDriverVersion),
+    (Get-FirefoxVersion),
+    (Get-GeckodriverVersion)
+)
+
+$markdown += New-MDHeader ".NET Core SDK" -Level 3
+$markdown += New-MDList -Style Unordered -Lines @(
+    (Get-DotNetCoreSdkVersions)
+)
+
+$markdown += New-MDHeader "Az Module" -Level 3
+$markdown += New-MDList -Style Unordered -Lines @(
+    (Get-AzModuleVersions)
```

Comment:
Looks like something went wrong with this section
![image](https://user-images.githubusercontent.com/16715858/93304386-9ebe0680-f805-11ea-8949-c51d976f104f.png)